### ISSUE TITLE
issue: Order of test cases after 9 cases fail 

### ISSUE THREAD

**Issue by LuisMBaezCo** [searleser97/cpbooster#56]
```shell
❯ cpb --version
2.6.3
```

I want to report the following problem:
I am testing cpbooster for multiple cases, more than 9, and when I run the command `cpb test ...` the order of the test cases is failing.

```diff
❯ cpb test E_BreakingtheWall.cpp
Compiling...

++Test Case 1:  A C 

 Input 

5
20 10 30 10 20


 Your Output 

10

-- Test Case 10:  A C 

 Input 

7
3 9 13 15 18 5 16


  Your Output   | Correct Answer 
                |                
4               |5                 


-- Test Case 11:  A C 

 Input 

8
12 18 14 2 10 18 14 13


  Your Output   | Correct Answer 
                |                
4               |5                 


-- Test Case 12:  A C 

 Input 

5
7 14 7 2 20


  Your Output   | Correct Answer 
                |                
3               |4                 


++Test Case 2:  A C 

 Input 

3
1 8 1


 Your Output 

1

Test Case 3:  A C 

 Input 

6
7 6 6 8 5 8


 Your Output 

4

Test Case 4:  A C 

 Input 

6
14 3 8 10 15 4


 Your Output 

4

Test Case 5:  A C 

 Input 

4
1 100 100 1


 Your Output 

2

Test Case 6:  A C 

 Input 

3
40 10 10


 Your Output 

7

Test Case 7:  A C 

 Input 

10
3 10 12 10 11 13 10 4 10 11


 Your Output 

4

Test Case 8:  A C 

 Input 

17
52 54 65 51 79 94 94 96 29 92 85 76 97 91 51 45 65


 Your Output 

32

Test Case 9:  A C 

 Input 

6
70 81 86 84 29 18


 Your Output 

16
```

**Comment by searleser97** [comment]
:o thanks for the bug report will definitely take a look at this one, I believe we will have to sort them after reading them from the directory

**Comment by searleser97** [comment]
This has been fixed in `2.6.4` https://github.com/searleser97/cpbooster/releases/tag/v2.6.4, please update by just running `npm install cpbooster -g`
